filament/src/details/Camera.h

```diff
@@ -73,13 +81,19 @@ class FCamera : public Camera {
 
     // viewing the projection matrix to be used for rendering, contains scaling/shift and possibly
     // other transforms needed by the shaders
-    math::mat4 getProjectionMatrix() const noexcept;
+    math::mat4 getProjectionMatrix(uint8_t eye = 0) const noexcept;
 
     // culling the projection matrix to be used for culling, contains scaling/shift
     math::mat4 getCullingProjectionMatrix() const noexcept;
 
+    math::mat4 getEyeFromViewMatrix(int eye) const noexcept { return mEyeFromView[eye]; }
```

Comment:
sometimes `eye` is a `uint8_t` and sometimes it's a `int`. We should be consistant. I think it should be unsigned at least.